cosigner/cosigner_test.go

```diff
@@ -45,7 +45,7 @@ func TestSimpleCosigner(t *testing.T) {
 	require.NoError(t, err, "failed to generate key pair")
 
 	cosignerClaims := pktoken.CosignerClaims{
-		Iss:         "example.com",
+		Issuer:      "example.com",
```

Comment:
@lgmugnier Good catch. Mind if I break this out into a separate smaller PR to get it in faster?

Reply:
sure, if you'd like. Is it going into a new versioned release? Or do you just want to get it into a different PR?

Reply:
I just broke it out into it's own, very small PR.

Normally I was say this should get a versioned release because it breaks downstream, but we are lucky in that no one depends on the Cosigner code yet.
https://github.com/openpubkey/openpubkey/pull/99